projects/extension/src/background/index.ts

```diff
@@ -173,6 +212,14 @@ window.uiInterface = {
     if (manager.state === "crashed") return manager.error
     else return undefined
   },
+  get wellKnownChainBootnodes() {
+    return {
+      [polkadot.id]: polkadot.bootNodes,
+      [ksmcc3.id]: ksmcc3.bootNodes,
+      [westend2.id]: westend2.bootNodes,
+      [rococo_v2_2.id]: rococo_v2_2.bootNodes,
```

Comment:
I don't understand this. Are the keys arrays of the object, or is that some weird JS syntax?

Reply:
Some [weird TS syntax](https://www.typescriptlang.org/docs/handbook/2/objects.html#index-signatures)

Reply:
This is due to [index signatures](https://www.typescriptlang.org/docs/handbook/2/objects.html#index-signatures).
Due to the fact that `ChainBootnodes` (which is the return of the [wellKnownChainBootnodes](https://github.com/paritytech/substrate-connect/pull/1079/files#diff-4de63d371461c5b51ea9a173eccc4c3b114374baa9388b564a3033e112f58924R215) has an indexed signature, and since the `ids` are strings they can be used  as indexes.
The following code is equivalent to the one above (just the one above is shorthanded):
```ts
    const obj: ChainBootnodes = {}
    obj[polkadot.id] = polkadot.bootNodes
    obj[ksmcc3.id] = ksmcc3.bootNodes
    obj[westend2.id] = westend2.bootNodes
    obj[rococo_v2_2.id] = rococo_v2_2.bootNodes
    return obj
```